TileFormats/Instanced3DModel/README.md

```diff
@@ -98,19 +121,51 @@ In either case, `header.gltfByteLength` contains the length of the glTF field in
 
 The `instances` field immediately follows the `glTF` field (which may be omitted when `header.gltfByteLength` is `0`).
 
-The `instances` field contains `header.instancesLength` of tightly packed instances.  Each instance has three fields:
+The `instances` field contains `header.instancesLength` of each instance field which are stored in groups of each field.
+
+Each instance has the following fields:
+
+| Field name | Data type | Description | Required |
+| --- | --- | --- | --- |
+| `position.x` | `uint16` | The x-coordinate in quantized Cartesian coordinates. | `yes` |
+| `position.y` | `uint16` | The y-coordinate in quantized Cartesian coordinates. | `yes` |
+| `position.z` | `uint16` | The z-coordinate in quantized Cartesian coordinates. | `yes` |
+| `encodedAxis.x` | `uint` | The first component of the oct-encoded `axis` vector. | `yes` |
+| `encodedAxis.y` | `uint` | The second component of the oct-encoded `axis` vector. | `yes` |
+| `encodedAngle` | `uint` | The angle to rotate about the axis from `0` to `2π` scaled to fill a single byte. | `yes` |
+| `batchId` | `uint16` | ID in the range `[0, length of arrays in the Batch Table)`, which indicates the corresponding properties. | `if header.batchTableByteLength > 0`
+
+### Position for Translation
+
+`position.x`, `position.y`, and `position.z` are stored as `uint16` positions in the [quantized volume](#quantized-volume) defined by the `offset` and `scale` fields in the header.
+
+Transforming `position` in instance region space to `position_m` in model space can be done using the formula:
+
+`position_m` = `offset` + [`position` * `scale` / (`2^16-1`)].
+
+### Axis and Angle for Rotation
+
+`encodedAxis` is stored as two components in oct-encoded format as described in
+[*A Survey of Efficient Representations of Independent Unit Vectors* by Cigolle et al.](http://jcgt.org/published/0003/02/01/)
+The [AttributeCompression](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Core/AttributeCompression.js)
+module in Cesium contains an implementation for encoding and decoding vectors in this fashion.
+
+`encodedAngle` is the rotation about the axis, from `0` to `2π` scaled to a range of `0` to `255`. It can be decoded to using the formula:
+
+`decodedAngle = encodedAngle * 2π / 255.0`.
 
-|Field name|Data type|Description|
-|----------|---------|-----------|
-| `longitude` | `double` | The longitude, in radians, in the range `[-PI, PI]`. |
-| `latitude` | `double` | The latitude, in radians, in the range `[-PI / 2, PI / 2]`. |
-| `batchId` | `uint16`  | ID in the range `[0, length of arrays in the Batch Table)`, which indicates the corresponding properties. |
+This format can be easily converted to a quaternion or [a rotation matrix](#creating-a-rotation-matrix-for-an-instance).
 
-_TODO: make this much more memory efficient and more general, [#33](https://github.com/AnalyticalGraphicsInc/3d-tiles/issues/33)._
+### Scaling using TILES3D_SCALE and TILES3D_NON_UNIFORM_SCALE
```

Comment:
Actually the way I have it now is that they are both applied if present. So you can have a non-uniform scale and a uniform scale and they get applied on top of each other. Order doesn't matter for multiplication since they're in Jordan form.